types/levelup/tslint.json

```diff
@@ -8,6 +8,10 @@
                     [
                         "NeedsExportEquals",
                         false
+                    ],
+                    [
+                        "NoDefaultExport",
```

Comment:
Why was this needed?

Reply:
I explained it in the comment above (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57663#issuecomment-991283057)

Do correct me if I am mistaken!

Reply:
Sorry, I did miss your comment.  Still, when I change this file to the preferred form (just `extends`) it seems to pass?

Regardless, as the bot [mentioned](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57663#issuecomment-991160857), it seems like the exports of this package are inconsistent with those of the underlying library?

Reply:
You are correct - also according to the comment from [Martin-badin](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/57663#discussion_r767104470) - the default export has been removed in version 5.0.
I will continue to bring it in line with the latest version of the library.